src/topology/algebra/valued_field.lean

```diff
@@ -231,7 +231,7 @@ lemma valued.continuous_extension : continuous (valued.extension : hat K → Γ
         conv {congr, skip, skip, rw ← (one_mul (1 : hat K))},
         refine tendsto.mul continuous_fst.continuous_at
                            (tendsto.comp _ continuous_snd.continuous_at),
-        convert topological_division_ring.continuous_inv (1 : hat K) zero_ne_one.symm,
+        convert @continuous_at_inv₀ (hat K) _ _ _ _ _ zero_ne_one.symm,
```

Comment:
This might be nicer as something like
```suggestion
        convert continuous_at_inv₀ (zero_ne_one.symm : 1 ≠ (0 : hat K)),
```
(untested)

Reply:
ha, yeah, I had that and then changed it.